src/EasyAbp.Abp.SettingUi.Web/Pages/SettingManagement/SettingUi/Index.js

```diff
@@ -46,23 +46,35 @@
         })
     }
 
-    $('#tabs-nav .nav-item .nav-link').click(function () {
-        var _this = $(this);
-        if (_this.attr('data-bs-target') !== undefined) {
-            return;
-        }
+    var bind_nav_event_fn = function (element) {
         var btn_set = setInterval(function () {
-            var _id = '#' + _this.data('id') + " ";
+            var _id = '#' + element.data('id') + " ";
             event_fn(_id);
             if ($(_id + " form").length > 0) {
                 clearInterval(btn_set);
+            } else {
```

Comment:
The `else` statement may not be necessary

Reply:
I don't think it always needs to be executed. It also clear the interval when the form is found.